app/models/project.rb

```diff
@@ -0,0 +1,66 @@
+class Project < ApplicationRecord
+
+    belongs_to :user
+    has_many :lead
+    has_many :leads, through: :lead, source: :user
+    
+    extend FriendlyId 
+
+    friendly_id :title, use: :slugged
```

Comment:
user more than title with this, because if the title is duplicate it will user a timestap into the slug and it will see bad or extrange. Add project_type or city.

Reply:
Done